src/main/java/mate/academy/Main.java

```diff
@@ -2,17 +2,23 @@
 
 import java.time.LocalDate;
 import java.time.LocalDateTime;
+import mate.academy.exception.AuthenticationException;
+import mate.academy.exception.RegistrationException;
+import mate.academy.lib.Injector;
 import mate.academy.model.CinemaHall;
 import mate.academy.model.Movie;
 import mate.academy.model.MovieSession;
+import mate.academy.service.AuthenticationService;
 import mate.academy.service.CinemaHallService;
 import mate.academy.service.MovieService;
 import mate.academy.service.MovieSessionService;
 
 public class Main {
-    public static void main(String[] args) {
-        MovieService movieService = null;
+    private static final Injector injector = Injector.getInstance("mate.academy");
 
+    public static void main(String[] args) throws RegistrationException, AuthenticationException {
+        MovieService movieService =
+                (MovieService) injector.getInstance(MovieService.class);
```

Comment:
Hereinafter it's better to move instances:
```suggestion
    private static final Injector injector = Injector.getInstance("mate.academy");
    private static final MovieService movieService =
                (MovieService) injector.getInstance(MovieService.class);

    public static void main(String[] args) throws RegistrationException, AuthenticationException {
```